apps/admin/backend/src/store.ts

```diff
@@ -1276,19 +1315,39 @@ export class Store {
           cvrs.batch_id as batchId,
           scanner_batches.scanner_id as scannerId,
           cvrs.sheet_number as sheetNumber,
-          cvrs.votes as votes
+          cvrs.votes as votes,
+          aggregated_adjudications.adjudications as adjudications
         from cvrs 
         inner join scanner_batches on cvrs.batch_id = scanner_batches.id
         inner join ballot_styles on
           cvrs.election_id = ballot_styles.election_id and 
           cvrs.ballot_style_id = ballot_styles.id
+        left join (
+          select
+            cvr_id,
+            case when id is null 
```

Comment:
Ah, this is the primary key. Good catch. This is a hold-over from when I was left joining first and grouping second, rather than grouping and then left joining. In the original case, I had to prevent the `json_group_array` from treating `null` as a member in the array. Will remove in a commit related to the choice of primary key you point out in another comment.